lib/wcc/contentful/model.rb

```diff
@@ -5,20 +5,86 @@ class WCC::Contentful::Model
   extend WCC::Contentful::Helpers
   extend WCC::Contentful::ModelValidators
 
+  # The Model base class maintains a registry which is best expressed as a
+  # class var.
+  # rubocop:disable Style/ClassVars
+
   class << self
+    ##
+    # The configured store which executes all model queries against either the
+    # Contentful CDN or a locally-downloaded copy.
+    #
+    # See the {sync_store}[rdoc-ref:WCC::Contentful::Configuration.sync_store] parameter
+    # on the WCC::Contentful::Configuration class.
     attr_accessor :store
   end
 
+  @@registry = {}
+
   def self.all_models
+    # TODO: this needs to use the registry but it's OK for now cause we only
```

Comment:
doing this now? had this thought before seeing your comment

Reply:
I'm not really sure what the use case is for even having this method honestly, it's really been just a hack for me to clear out specs.  I'd be for getting rid of the method entirely.